### PR TITLE
S1 t3 instructions

### PR DESCRIPTION
**Pull request by daniellaskinner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
...
+# 2020-microHippos-pairs game
+
+Game instructions:
+
+The aim of the game is to find all the matching pairs,
+Click on two images to see the devs who are there.
+Each turn, try and remember the devs places,
+When you have found a matching pair you will still see their faces.
+
+If pairing all the devs is done, then the game is won!
```
Comment (markandmarkup) [inline README.md]: Suggest - "Then the *matching* game is won"
Keeps the even beat of the line ;)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
-# year-cohort-projectName
-project template repo
+# 2020-microHippos-pairs game
+
+Game instructions:
+
+The aim of the game is to find all the matching pairs,
+Click on two images to see the devs who are there.
```
Comment (markandmarkup) [inline README.md]: Click on two 'cards' or 'squares' - it's a little clearer I think.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,10 @@
...
+Each turn, try and remember the devs places,
```
Comment (markandmarkup) [inline README.md]: Each turn, try to remember their places

**Review by markandmarkup — COMMENTED** [review]
Just a few tweaks :D

**Review by paulms1 — APPROVED** [review]

**Pull request merged**